src/web-ui/src/partials/AppModal/DemoWalkthrough/DemoWalkthroughPageLayout.vue

```diff
@@ -0,0 +1,240 @@
+<template>
+  <div class="layout">
+    <main :class="{ main: true, 'main--mobile': isMobile }" ref="main">
+      <slot></slot>
+    </main>
+
+    <div
+      v-if="showNav"
+      :class="{ 'demo-walkthrough-nav d-flex justify-content-between': true, 'demo-walkthrough-nav--mobile': isMobile }"
+    >
+      <button v-if="!isLast" class="end-tour btn btn-outline-primary" @click="skip">End Tour</button>
+
+      <div :class="{ 'ml-auto': isLast }">
+        <button class="prev btn btn-outline-primary" @click="prevTourPage">Previous</button>
+
+        <button v-if="isLast" class="end-tour btn btn-primary ml-2" @click="skip">End Tour</button>
+        <button v-else class="next btn btn-primary ml-2" @click="nextTourPage">Next</button>
+      </div>
+    </div>
+
+    <svg ref="svg" style="position: absolute; left: 0; width: 100%; top: 0; height: 100%; pointer-events: none;">
+      <defs>
+        <marker id="arrow" refX="1.5" refY="1.5" markerWidth="2" markerHeight="3" orient="auto">
+          <path d="M 0 0 L 2 1.5 L 0 3 z" />
+        </marker>
+      </defs>
+
+      <template v-if="arrowPoints">
+        <polyline
+          v-for="(points, i) in arrowPoints"
+          :key="i"
+          :points="`${points.map(([x, y]) => `${x}, ${y}`).join(' ')}`"
+          fill="none"
+          class="arrow-line"
+          stroke-width="5"
+          stroke-linejoin="round"
+          marker-end="url(#arrow)"
+        />
+      </template>
+    </svg>
+  </div>
+</template>
+
+<script>
+import { mapActions, mapState } from 'vuex';
+
+import { APP_MODAL_ID } from '../config';
+
+export default {
+  name: 'DemoWalkthroughPageLayout',
+  props: {
+    showNav: { type: Boolean, default: true },
+    isLast: { type: Boolean, default: false },
+    // Side: 'TOP' | 'RIGHT' | 'BOTTOM' | 'LEFT'
```

Comment:
Lets remove the commented code if not needed

Reply:
It's documentation for the types, not commented code